maven-embedder/src/test/java/org/apache/maven/cli/CLIManagerDocumentationTest.java

```diff
@@ -100,7 +101,7 @@ public String getOptionsAsHtml()
     public void testOptionsAsHtml()
         throws IOException
     {
-        File options = getTestFile( "target/test-classes/options.html" );
+        File options = new File( "target/test-classes/options.html" );
```

Comment:
Why this change?

Reply:
These classes no longer extend `PlexusTestCase`, so they lose the `getTestFile` utility method, that [effectively does just this](https://github.com/codehaus-plexus/plexus-containers/blob/dfb1d7132c7f52122a6e18da8a9fb86e7592dee2/plexus-container-default/src/main/java/org/codehaus/plexus/PlexusTestCase.java#L229).

Reply:
Looking at `getBasedir()` won't this fail is `-Dbasedir` is set and you don't read it?

Reply:
In this case, the test file is stored in that specific location, so setting `-Dbasedir` to anything else would cause the existing test to fail! It doesn't look like `basedir`'s being set anywhere for these tests, so that behaviour may not need to be carried over.